### PR TITLE
Hubert / adding contracts for symbol sandbox

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: examples/contracts_for_symbol/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  if (data.msg_type === "contracts_for") {
+    console.log(data.contracts_for);
+  }
+
+  connection.removeEventListener("message", contractsForSymbolResponse, false);
+};
+
+const getContractsForSymbol = async () => {
+  token = localStorage.getItem("token1");
```
Comment (akmal-binary) [inline examples/contracts_for_symbol/index.js:39]: Suggestion: how about we give token1 a more meaningful name?

File: examples/contracts_for_symbol/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api/dist/DerivAPIBasic";
```
Comment (mahdiyeh-fs) [inline examples/contracts_for_symbol/index.js:1]: better to add a empty line after imports

Comment (hubert-deriv) [reply examples/contracts_for_symbol/index.js:1]: fixed 👍

File: examples/contracts_for_symbol/index.js (hunk 1 above)
Comment (hubert-deriv) [reply examples/contracts_for_symbol/index.js]: I renamed it to login_token, as you receive the token through the URL that is saved in the localstorage after logging in (OAuth).

**Review by mahdiyeh-fs — APPROVED** [review]

**Review by adam-deriv — APPROVED** [review]

**Pull request merged**